docs/sources/flow/getting-started/collect-prometheus-metrics.md

```diff
@@ -117,3 +120,148 @@ prometheus.scrape "example" {
 
 For more information on configuring metrics delivery, refer to
 [prometheus.remote_write][].
+
+## Collect metrics from Kubernetes Pods
+
+Grafana Agent Flow can be configured to collect metrics from Kubernetes Pods
+by:
+
+1. Discovering Kubernetes Pods to collect metrics from
+2. Collecting metrics from those discovered Pods
+
+To collect metrics from Kubernetes Pods, complete the following steps:
+
+1. Follow [Configure metrics delivery](#configure-metrics-delivery) to ensure
+   collected metrics can be written somewhere.
+
+2. Discover Kubernetes Pods:
+
+    1. Add the following `discovery.kubernetes` component to your configuration
+       file:
+
+       ```river
+       discovery.kubernetes "DISCOVERY_LABEL" {
+         role = "pod"
+       }
+       ```
+
+       This will generate one Prometheus target for every exposed port on every
+       discovered Pod.
```

Comment:
Nit: it's kinda implied that this gets _all_ pods it can reach, right? Do we also want to spell it out?